### DISCUSSION TITLE
Question about extends and schedule presents

### DISCUSSION THREAD

**Discussion by kjq** [renovatebot/renovate#19599] [Request Help] [answered]
Probably a dumb question but, if I extend a base configuration that has a schedule preset and then in the project configuration add to extends with another schedule preset - does the extended one override the original scheduled preset or append to it?

**default.json:**

```json
    "extends": [
        "config:base",
        ":automergeMinor",
        ":automergePr",
        ":combinePatchMinorReleases",
        ":prImmediately",
        ":rebaseStalePrs",
        "schedule:nonOfficeHours"
    ],
```

**Another project or rule:**

```json
  "extends": [
    "local>operations/renovate",
    "schedule:weekly"
  ]
```

Will "weekly" replace "nonOfficeHours" or will they both run?

**Comment by rarkins** [comment, marked as answer]
It should replace

**Reply by kjq** [reply]
Thanks!
